### PR TITLE
Ft/unit test

### PR DESCRIPTION
**Pull request by Traineau**

### CODE REVIEW

File: helpers/tools_test.go [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+	g := NewGomegaWithT(t)
+
+	stringToUint, err := ParseUInt64("5")
+
+	if err != nil {
+		fmt.Errorf("could not parse string to int")
+	}
+	
+	g.Expect(stringToUint).To(BeIdenticalTo(uint64(5)), "String should have become a int")
+}
```
Comment (Araknyfe) [inline helpers/tools_test.go]: blank line :)

File: strategies/exportContext_test.go [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+	}
+
+	b, err := context.Export(articles)
+	if err != nil {
+		log.Printf("couldn't export: %v", err)
+		return
+	}
+
+	g.Expect(b).ShouldNot(BeNil(), "Articles should have transformed to Bytes")
+}
```
Comment (Araknyfe) [inline strategies/exportContext_test.go]: blank line :)

File: strategies/exportCsv_test.go [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+		data = append(data, articleArr)
+	}
+
+	err := writer.WriteAll(data)
+	if err != nil {
+		log.Printf("cannot write to file: %s", err)
+	}
+
+	g.Expect(b.Bytes()).ShouldNot(BeNil(), "Articles should have transformed to Bytes for CSV")
+}
```
Comment (Araknyfe) [inline strategies/exportCsv_test.go]: blank line :)

File: strategies/exportXls_test.go [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+	}
+	
+	row2 := sheet.AddRow()
+	row2.AddCell().Value = strconv.FormatUint(article.ID, 10)
+	row2.AddCell().Value = article.Title
+	row2.AddCell().Value = article.Header
+	row2.AddCell().Value = time.Time.String(article.CreatedOn)
+
+	g.Expect(sheet).ShouldNot(BeNil(), "Sheet should have been created")
+}
```
Comment (Araknyfe) [inline strategies/exportXls_test.go]: blank line :)

**Review by Araknyfe — APPROVED** [review]
LGTM :)

**Review by AlexandreLch — APPROVED** [review]

**Pull request merged**